code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -136,10 +136,31 @@
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/plasma/can_attach(obj/mecha/working/M)
 	if(..()) //combat mech
-		return 1
+		return TRUE
 	else if(M.equipment.len < M.max_equip && istype(M))
-		return 1
-	return 0
+		return TRUE
+	return FALSE
+
+/obj/item/mecha_parts/mecha_equipment/weapon/energy/pka_carbine
+	equip_cooldown = 30
+	name = "Proto-Kinetic carbine"
+	desc = "An high pressure kinetic accelerator which shoots bursts of kinetic force in rapid succession."
+	icon_state = "mecha_scatter"
+	energy_drain = 60
+	projectile = /obj/item/projectile/kinetic/mech_carbine
+	fire_sound = 'sound/weapons/kenetic_accel.ogg'
+	harmful = TRUE
+	projectiles_per_shot = 3

```

Comment:
I'd rather this be 1, since the endgoal is to attach upgrades to this.

This will absolutely demolish mobs if you add the aoe damage.

Reply:
but it looks cooler when triple shotting

Reply:
lol